website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -350,6 +354,10 @@ The following attributes are exported:
 
 * `fqdn` - The FQDN of the Azure Kubernetes Managed Cluster.
 
+* `private_fqdn` - The FQDN when private link has been enabled with `private_link_enabled`, this name is only resolvable inside the AKS vnet. 
```

Comment:
(same here) could we spell this out so it's a little clearer:

```suggestion
* `private_fqdn` - The FQDN for the Kubernetes Cluster when private link has been enabled, which is is only resolvable inside the Virtual Network used by the Kubernetes Cluster.
```

Reply:
Resolved in new commit